src/test/java/com/artipie/rpm/http/RpmUploadRequestTest.java

```diff
@@ -45,4 +47,23 @@ void returnsFileNameKey() {
         );
     }
 
+    @ParameterizedTest
+    @CsvSource({
+        "/file.rpm?override=true,true",
+        "/file.rpm?some_param=true&override=true,true",
```

Comment:
@olegmoz I'd suggest to add `/file.rpm?some_param=false&override=true,true`

Reply:
@olenagerasimova makes sense, added this case